src/components/analytics/index.ts

```diff
@@ -1,94 +0,0 @@
-import { useWeb3React } from '@web3-react/core'
-import { useEffect } from 'react'
-import { UaEventOptions } from 'react-ga4/types/ga4'
-import { useLocation } from 'react-router-dom'
-import { isMobile } from 'utils/userAgent'
-import { getCLS, getFCP, getFID, getLCP, Metric } from 'web-vitals'
-
-import GoogleAnalyticsProvider from './GoogleAnalyticsProvider'
-
-const GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY = 'ga_client_id'
-const GOOGLE_ANALYTICS_ID: string | undefined = process.env.REACT_APP_GOOGLE_ANALYTICS_ID
-
-const storedClientId = window.localStorage.getItem(GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY)
-
-const googleAnalytics = new GoogleAnalyticsProvider()
-
-export function sendEvent(event: string | UaEventOptions, params?: any) {
-  return googleAnalytics.sendEvent(event, params)
-}
-
-export function outboundLink(
-  {
-    label,
-  }: {
-    label: string
-  },
-  hitCallback: () => unknown
-) {
-  return googleAnalytics.outboundLink({ label }, hitCallback)
-}
-
-export function sendTiming(timingCategory: any, timingVar: any, timingValue: any, timingLabel: any) {
-  return googleAnalytics.gaCommandSendTiming(timingCategory, timingVar, timingValue, timingLabel)
-}
-
-if (typeof GOOGLE_ANALYTICS_ID === 'string') {
-  googleAnalytics.initialize(GOOGLE_ANALYTICS_ID, {
-    gaOptions: {
-      storage: 'none',
-      storeGac: false,
-      clientId: storedClientId ?? undefined,
-    },
-  })
-  googleAnalytics.set({
-    anonymizeIp: true,
-    customBrowserType: !isMobile
-      ? 'desktop'
-      : 'web3' in window || 'ethereum' in window
-      ? 'mobileWeb3'
-      : 'mobileRegular',
-  })
-} else {
-  googleAnalytics.initialize('test', { gtagOptions: { debug_mode: true } })
-}
-
-const installed = Boolean(window.navigator.serviceWorker?.controller)
-const hit = Boolean((window as any).__isDocumentCached)
-const action = installed ? (hit ? 'Cache hit' : 'Cache miss') : 'Not installed'
-sendEvent({ category: 'Service Worker', action, nonInteraction: true })
-
-function reportWebVitals({ name, delta, id }: Metric) {
```

Comment:
Are we capturing web vitals through amplitude?

Reply:
I think this is already captured by Amplitude. Could you just verify?

Reply:
yup confirmed